java-vision/google-cloud-vision/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,4811 @@
+[
```

Comment:
These configs were generated by the tracing agent for java-vision unit tests and integration tests.